main.py

```diff
@@ -39,18 +40,36 @@ def write_pages(filename, pages):
         pdf_writer.write(output_file)
 
 
-# TODO: Take input and output file paths as flags,
+def parse_arguments():
+    parser = argparse.ArgumentParser(prog="Book Maker",
+                                     description="Converts PDF into books for easy printing")
+    parser.add_argument("input_pdf",
+                        type=str,
+                        help="Input PDF filepath"
+                        )
+    parser.add_argument("-o",
+                        "--output_pdf",
+                        type=str,
+                        help="Output PDF filepath",
+                        default="output.pdf"
+                        )
+    args = parser.parse_args()
+    return args.input_pdf, args.output_pdf
```

Comment:
Return the entire `args` container here, to avoid having to add more elements here as the flags increase.